framework/src/modules/chain/loader.js

```diff
@@ -541,9 +537,9 @@ __private.getTransactionsFromNetwork = async function() {
 
 	const transactions = result.transactions;
 	try {
-		const {
-			transactionsResponses,
-		} = modules.processTransactions.validateTransactions(transactions);
+		const { transactionsResponses } = validateTransactions(exceptions)(
```

Comment:
in the process of validateTransactions, it calls the exception_handler functions, and it excepts to exists.
Maybe I should change the exceptions handler to accepts undefined keys?

Reply:
i was suggesting to pass an empty object here. It makes more sense because no transaction should be considered an exception when they are loaded from network.